### ISSUE TITLE
wifi during exam

### ISSUE THREAD

**Issue by brookie13** [nus-cs2030/2223-s1#315]
Just want to ask, during the exam are we allowed to connect to the school wifi? My concern is that docker requires internet connection and I plan on using vim for the exam.

**Comment by qwqdrz** [comment]
I think yes since we are allowed to vim codecrunch submission online. So that means connecting to wifi is allowed I guess

**Comment by wjiayis** [comment]
Yup, since the exam does not involve Examplify, our internet is not blocked.

**Comment by Grauda123** [comment]
yes we are allowed to check the java api online too during the exams

**Comment by dohaduong** [comment]
yep wifi is allowed but we are not allowed to access anything else except for java api

**Comment by ZYN2122** [comment]
We also can access codecrunch submissions I believe but not sure if we can access past submissions for labs etc

**Comment by loberyyq** [comment]
Yes I think we have wifi and we can access Java API and codecrunch
